packages/material-ui/src/test-utils/createMount.js

```diff
@@ -59,7 +59,9 @@ export default function createMount(options = {}) {
   mountWithContext.attachTo = attachTo;
   mountWithContext.cleanUp = () => {
     ReactDOM.unmountComponentAtNode(attachTo);
-    attachTo.parentNode.removeChild(attachTo);
+    if (attachTo.parentNode) {
```

Comment:
This means the test was never mounted in a full document which should rightfully throw. Not sure how this relates to moving files accross packages?

Reply:
This change enables repeated usage of cleanUp. It happens when there is an *after()* cleanup and a *describeConformance()* cleanup.

Reply:
revering.